cask-cli.el

```diff
@@ -224,6 +224,14 @@ directory called bin in the root directory.
 The output is formatted as a colon path."
   (princ (concat (s-join path-separator (cask-exec-path (cask-cli--bundle))) "\n")))
 
+(defun cask-cli/eval (&optional form)
+  (mapc (lambda (n) (add-to-list 'load-path n))
```

Comment:
Thought about it. Didn't bother because it's "cli" --- i.e. emacs is going to terminate anyway, which is as good a way to reset the load-path as a dynamic binding.

I've added it as a macro. If you add "eval", then someone is going to want "eval-file" before long:-)